assets/js/api/CustomTranslationApi.js

```diff
@@ -0,0 +1,34 @@
+import $ from 'jquery'
+
+export class CustomTranslationApi {
+  constructor (programSection) {
+    this.programSection = programSection
+  }
+
+  getCustomTranslation (programId, language, successCallback, errorCallback) {
+    $.ajax({
+      url: '../translate/custom/project/' + programId + '?field=' + this.programSection + '&language=' + language,
```

Comment:
Could we use router here instead, or does it take a bit too much research to get it to work now?

Reply:
I think it is okay for now and then we can create another PR that uses the router if we think it is worth doing that

Reply:
sure